src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Truck;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    List<Truck> list = new ArrayList<>();
+    Truck truck = new Truck();
+    @Override
+    public List<Truck> get() {
+        list.add(truck);
+        return list;
+    }
+}
```

Comment:
always make sure to leave one empty line at the end of the file to avoid this on github
![image](https://github.com/mate-academy/jv-pecs/assets/49767049/e8410c7a-c9ad-465d-b41e-b50ad9b6e276)